script_library/run-ansible.sh

```diff
@@ -1,9 +1,13 @@
 #!/bin/bash
 
-
+# exit if vars is not set
```

Comment:
I would still prefer to not exit the shell. I suppose this doesn't exit the shell but stop the process by returning an error, is that correct? In that case I am okay with it.

Reply:
yup, it will stop the process and print the message